source/app.d

```diff
@@ -1,5 +1,16 @@
 import vibe.d;
 
+/++
+ + Renders an error page, everytime an error occured
+ +/
+void displayError(HTTPServerRequest req, HTTPServerResponse res, HTTPServerErrorInfo error)
```

Comment:
Looks like you can just name the unused variables as `_` and it will get ignored by dscanner afaik.